src/main/java/com/bitquest/bitquest/BitQuest.java

```diff
@@ -811,6 +811,8 @@ public boolean onCommand(CommandSender sender, Command cmd, String label, String
                     } catch (IOException e1) {
                         e1.printStackTrace();
                     }
+                    if (sendAmount >= 20000)
+                    {
                     try {
```

Comment:
Running it with Docker is surprisingly easy and, I believe, the preferred method.  If you're on Windows it'll probably be unpleasant, but then again, what isn't on Windows? :P